tools/extract-apt-setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e -u
+
+cd "$(git rev-parse --show-toplevel)"
```

Comment:
Can you make this commit (5e14b9649f2db81261fe0e570ca1a5a14c260b96) have a more detailed commit message? Currently, it doesn't explain the rationale for moving (which is obvious at this point but would be worth clarifying with a single sentence), and implies that extract-apt-setup is unchanged, when you've actually made some changes to the script.

Reply:
The changes to the script are strictly necessary to make it work from its new location.

Reply:
```
tools: Move extract-apt-setup and debian-checksum to tools directory

This directory contains miscellaneous scripts which are useful for development.
- Rewrite extract-apt-setup to behave correctly in its new location
```